.github/workflows/quality-check.yaml

```diff
@@ -20,9 +20,10 @@ jobs:
   check-and-test:
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
```

Comment:
Just to check, presumably this wouldn't make testing much slower anyway, as gpflux tests are very fast; correct?

Reply:
The tests take just as long as before if they all pass. However, if one of them fails early on, then previously we would have cancelled all the others, while now we will wait until they all finish (around 10 minutes).